### PR TITLE
mgmt, firstbeta for devopsinfrastructure from TypeSpec

### PR DESCRIPTION
**Pull request by XiaofeiCao**
# Description

Issue: https://github.com/Azure/sdk-release-request/issues/5139
namespace review: https://github.com/Azure/azure-sdk-pr/issues/1908

Strangely pipeline hang at generating... Retried few times, hang still. Figuring out the cause.. Use manual generation this time.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3813389&view=results

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by haolingdong-msft — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -40,6 +40,7 @@
     <module>sdk/botservice</module>
     <module>sdk/changeanalysis</module>
     <module>sdk/chaos</module>
+    <module>sdk/clientcore</module>
```
[the remark this answers is not in the document]
Comment (XiaofeiCao) [reply pom.xml:43]: No idea.. Don't know why it got changed, though I think current sequencing should be correct.

File: sdk/devopsinfrastructure/azure-resourcemanager-devopsinfrastructure/src/main/java/com/azure/resourcemanager/devopsinfrastructure/models/ManagedServiceIdentity.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+     */
+    @JsonProperty(value = "type", required = true)
+    private ManagedServiceIdentityType type;
+
+    /*
+     * The identities assigned to this resource by the user.
+     */
+    @JsonProperty(value = "userAssignedIdentities")
+    @JsonInclude(value = JsonInclude.Include.NON_NULL, content = JsonInclude.Include.ALWAYS)
+    private Map<String, UserAssignedIdentity> userAssignedIdentities;
```
Comment (XiaofeiCao) [inline sdk/devopsinfrastructure/azure-resourcemanager-devopsinfrastructure/src/main/java/com/azure/resourcemanager/devopsinfrastructure/models/ManagedServiceIdentity.java:41]: Seems correct.

**Pull request merged**